txauthor.go

```diff
@@ -35,12 +35,17 @@ func (tx *TxAuthor) SetSourceAccount(accountNumber int32) {
 	tx.sendFromAccount = uint32(accountNumber)
 }
 
-func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool) {
+func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool, wallet *Wallet) bool {
```

Comment:
Please revisit the first point in https://github.com/raedahgroup/dcrlibwallet/pull/117#discussion_r395984252